packages/migrate-config/tests/migrate-config-cli.test.js

```diff
@@ -83,6 +84,7 @@ describe("@eslint/migrate-config", async () => {
 			// run the migration for mjs
 			execSync(
 				`node src/migrate-config-cli.js ${filePath} ${gitignoreFlag}`,
+				{ stdio: "inherit" },
```

Comment:
Is this change intentionally committed?

Reply:
Yes. Otherwise the errors are swallowed during testing.